login-fire.html

```diff
@@ -84,25 +84,23 @@
     }
 
     .emailPWError {
-      color: #dd2c00;
-      direction: ltr;
+      color: var(--error-color);
```

Comment:
Is `--error-color`? Sorry I thought you were creating these variables with no default.